library/common-library/templates/_agent-config.tpl

```diff
@@ -5,13 +5,26 @@ This helper should return the defaults that all agents should have
 {{- if include "common.verboseLog" . }}
 verbose: 1
 {{- end }}
+
 {{- if (include "common.nrStaging" . ) }}
 staging: true
 {{- end }}
+
+{{- with include "common.license._licenseKey" . }}
+{{- /* We will mostly override this using a environment variable but we can have it here just in case */}}
+license: {{ . }}
+{{- end }}
+
```

Comment:
Not a big fan of doing things in two different ways. Plus, this will cause the licenseKey to almost surely end up in a ConfigMap rather than a Secret. I'd stick with the env+secret approach for the LK.

Reply:
LGTM. Removing.

Reply:
Removed in [#f629333](https://github.com/newrelic/helm-charts/pull/757/commits/f6293332c3ba99e6898c337a9b5ee93c24f67c12)